File: AirTECWebAPI/Models/Passenger.cs
using System;
using System.Collections.Generic;

namespace AirTECWebAPI.Models;

public partial class Passenger
{
    public int Idpassenger { get; set; }

    public string Fullname { get; set; } = null!;

    public int? Phonenumber { get; set; }

    public virtual ICollection<Suitcase> Suitcases { get; set; } = new List<Suitcase>();

    public virtual ICollection<Ticket> Tickets { get; set; } = new List<Ticket>();

    public virtual ICollection<User> Users { get; set; } = new List<User>();
}

File: frontend/src/app/Interfaces/passenger.ts
export class Passenger {

    idpassenger?: number;
    fullname?: string;
    phonenumber?: number;
}

export class Suitcase {
    idsuitCase?: number;
    weight?: number;
    color?: string;
    idpassenger?: number;
}
export class User {

    iduser?: number;
    fullname?: string;
    phonenumber?: number;
    password?: string;
    email?: string;
    idpassenger?: number;
}